bin/MorphingMSSMvsSM.cpp

```diff
@@ -972,7 +1080,7 @@ int main(int argc, char **argv) {
   // Adding bin-by-bin uncertainties
   if (use_automc) {
     std::cout << "[INFO] Adding bin-by-bin uncertainties.\n";
-    cb.SetAutoMCStats(cb, 0.0);
+    cb.SetAutoMCStats(cb, 10.);
```

Comment:
Do we want to change the procedure of the autoMCStats? Is there a specific motivation to set the threshold value to 10 or is it more or less a number that looks reasonable to you?

Reply:
This is a question for Danny.

Reply:
autoMC stats approximates the bbb uncertainties as gaussian but this is not well motivated for events with a small number of effective events. A threshold of 10 is well motivated as it where the Gaussian distribution becomes a reasonable approximation for the Poisson distribution. With this option we will revert to the old treatment of bbb's (1 bbb per process per bin) with Poisson constraints for events with fewer than 10 effective events (i.e relative uncertainty < 1/sqrt(10)) - this will probably only affect some of the higher mass bins but since these are the most important bins for the high mass search and the dominant systematic uncertainties are bbb's it is well motivated to apply this threshold. P.S depending on how we choose the autoRebin threshold we might not end up with any such bins in the end (e.g if we set the maximum bbb threshold to 30%) but this might impact the sensitivity